api/mail.go

```diff
@@ -168,6 +169,10 @@ func (a *API) GenerateLink(w http.ResponseWriter, r *http.Request) error {
 		return internalServerError("User serialization error").WithInternalError(err)
 	}
 	resp["action_link"] = url
```

Comment:
i think we should also rename this to `verification_link` but let's do this in v2 instead because it's gonna be a breaking change